drivers/xen/events.c

```diff
@@ -218,7 +218,7 @@ static void events_isr(void *data)
 	 */
 	vcpu->evtchn_upcall_pending = 0;
 
-	compiler_barrier();
+	__DMB();
```

Comment:
We are trying to move away from `__XXX`. Can you use `dmb()` instead?

Reply:
Sure, thanks for review! Fixed.